core/src/main/java/org/keycloak/protocol/oidc/representations/OIDCConfigurationRepresentation.java

```diff
@@ -169,6 +169,9 @@
     @JsonProperty("pushed_authorization_request_endpoint")
     private String pushedAuthorizationRequestEndpoint;
 
```

Comment:
Could you also add a `boolean authorization_details_supported`?
as of: https://datatracker.ietf.org/doc/html/draft-ietf-oauth-rar-04#section-9
```
  The AS advertises support for "authorization_details" using the
   metadata parameter "authorization_details_supported" of type boolean.
```

Reply:
I will add this parameter.